dockerize/dockerize.py

```diff
@@ -8,26 +13,28 @@
 import shutil
 import subprocess
 import tempfile
+from dockerize.depsolver import DepSolver
```

Comment:
Do you know if there are any issues with this approach in Python 3? I don't recall exactly why I changed it (to the long form) but I believe it may have been to ensure compatibility and consistent behavior across all versions. If it's not a problem then I'll go ahead and commit the short form (i.e. `from .depsolver import DepSolver`).